src/monitor/Monitor.ts

```diff
@@ -939,7 +956,8 @@ export class Monitor {
   private async _getDecimals(decimalrequests: { chainId: number; token: string }[]): Promise<number[]> {
     return await Promise.all(
       decimalrequests.map(async ({ chainId, token }) => {
-        if (token === ZERO_ADDRESS) {
+        const ethAddressForChain = CONTRACT_ADDRESSES[chainId]?.eth?.address ?? ZERO_ADDRESS;
```

Comment:
Good candidate for `getEthAddressForChain`?
```suggestion
        const ethAddressForChain = getEthAddressForChain(chainId);
```

Reply:
ah shoot yeah good catch